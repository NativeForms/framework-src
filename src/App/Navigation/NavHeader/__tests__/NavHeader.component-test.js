import React from 'react';
import { shallow } from 'enzyme';

// app
import NavHeaderComponent from '../NavHeader.component';

const props = {
  scene: {},
  getScreenDetails: () => ({ options: {} }),
  onToggleMenu: jest.fn()
};

test('NavHeader should render correctly', () => {
  const wrapper = shallow(<NavHeaderComponent {...props} />);
  expect(wrapper).toMatchSnapshot();
});
