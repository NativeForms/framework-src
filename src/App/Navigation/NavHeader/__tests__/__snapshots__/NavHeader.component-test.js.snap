// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavHeader should render correctly 1`] = `
<Styled(Header)>
  <Styled(Left)>
    <Styled(Button)
      onPress={[Function]}
      transparent={true}
    >
      <Styled(Icon)
        name="menu"
      />
    </Styled(Button)>
  </Styled(Left)>
  <Styled(Body)>
    <Styled(Title) />
  </Styled(Body)>
  <Styled(Right) />
</Styled(Header)>
`;
